crates/polars-ops/src/series/ops/index.rs

```diff
@@ -32,7 +34,7 @@ where
     })
 }
 
-pub fn convert_to_index(s: &Series, target_len: usize) -> PolarsResult<IdxCa> {
+pub fn convert_to_positive_index(s: &Series, target_len: usize) -> PolarsResult<IdxCa> {
```

Comment:
*Technically* the term is non-negative 😄

Reply:
Haha.. it makes you happy if I rename it right?

Reply:
Maybe just `convert_to_unsigned_index`?